zivid_camera/src/zivid_camera.cpp

```diff
@@ -475,11 +531,23 @@ sensor_msgs::ImageConstPtr ZividCamera::makeColorImage(const std_msgs::Header& h
   return msg;
 }
 
+sensor_msgs::ImageConstPtr ZividCamera::makeColorImage(const std_msgs::Header& header,
+                                                       const Zivid::Image<Zivid::RGBA8>& image)
+{
+  auto msg = boost::make_shared<sensor_msgs::Image>();
+  fillCommonMsgFields(*msg, header, image.width(), image.height());
+  msg->encoding = sensor_msgs::image_encodings::RGBA8;
+  msg->step = static_cast<uint32_t>(4 * image.width());
```

Comment:
Suggestion: give a name to the number `4` here. I guess it is the number of bytes corresponding to `R`, `G`, `B` and `A`, but would be nice to have it explicit.

Reply:
can do